docs/DeveloperGuide.md

```diff
@@ -548,19 +514,24 @@ Those without any stars are user stories that were considered but will not be im
 **Extensions**
 
 * 1a. The list is empty.
-
   Use case ends.
 
-
 * 2a. The given index is invalid.
-
     * 2a1. System shows an error message prompting the user to enter a valid index.
-
       Use case resumes at step 1.
 
+### **Use case: UC19 — Delete Wedding**
```

Comment:
i'm actually not sure if deletion of wedding, task and tag should be said to be similar as deleting a contact because the extension 2a where the given index is invalid would not be directly applicable

Reply:
Ooh good catch -- I  can change that to "the given _contact_ is invalid" then the following 3 should be okay